docs/sphinx/basics/wfn_examples.rst

```diff
@@ -0,0 +1,157 @@
+.. _`sec:wfn_examples`:
+
+Examples of Forte inputs
+========================
+
+.. sectionauthor:: Francesco A. Evangelista
+
+Computing one or more solutions of a given symmetry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first example shows how to perform separate computations on different electronic
+states of methylene. We compute the lowest :math:`^3B_1` state and the first two :math:`^1A_1` states.
+All of these computations use ROHF orbitals optimized for the lowest :math:`^3B_1`.
+Here we use the python API interface of Forte, but it is easy to translate
+these examples to a psi4 psithon input.
+
+This input starts with the geometry specification::
+
+    import psi4
+    import forte
+
+    mol = psi4.geometry("""
+    0 3 # triplet state
+    C
+    H 1 1.085
+    H 1 1.085 2 135.5
+    """)
+
+    psi4.set_options({
+        'basis': 'DZ',
+        'scf_type': 'pk', # <-- request conventional two-electron integrals
+        'e_convergence': 12,
+        'reference' : 'rohf'} # <-- specify the reference
+        )
+
+After this, we set the input for the forte computation. We specify a multiplicity
+equal to 3 and the :math:`B_1` irrep (``root_sym = 2``).
+In this example e keep the lowest :math:`^A_1` MO doubly occupied
+(``'restricted_docc': [1, 0, 0, 0]``) and use an active space that contains three
+:math:`^A_1` MOs, and two MOs each of :math:`^B_1` and :math:`^B_2` symmetry::
+
+    psi4.set_module_options(
+    'FORTE', {
+        'active_space_solver' : 'fci',
+        'restricted_docc': [1, 0, 0, 0],
+        'active': [3, 0, 2, 2],           
+        'multiplicity': 3,            
+        'root_sym': 2, # <-- B1 symmetry
+    }
+    )
+    psi4.energy('forte')
+
+This input will run a CASCI computation (since we have not requested orbital optimization).
```

Comment:
Can you show an example of requesting orbital optimization, or link to one?

Reply:
Good point. Done